index.ts

```diff
@@ -402,9 +402,9 @@ export async function getLinkPreview(
   // Seems like fetchOptions type definition is out of date
   // https://github.com/node-fetch/node-fetch/issues/741
   const response = await fetch(fetchUrl, fetchOptions as any)
-    .then((res) => {
+    .then(async (res) => {
```

Comment:
the parent function is already async right? would you mind refactoring this to avoid using then?

Reply:
Done